marketplace_builder/views/pages/tutorials/custom-model-types/persisting-data-with-customizations.liquid

```diff
@@ -0,0 +1,118 @@
+---
+converter: markdown
+metadata:
+  title: Persisting Data With Customizations
+  description: Build your custom data structure without any limitations with Custom Model Types.
+slug: tutorials/custom-model-types/persisting-data-with-customizations
+searchable: true
+---
+
+Let's use the example to see exactly what is happening at each stage of Custom Model Type definition life cycle.
+
+## Step 1. Requirements
+
+I want to store my "todo list" in platformOS
+
+**Todo list:**
+
+| Task                 | Priority |
+|----------------------|----------|
+| Clean the apartment  | Low      |
+| Do the homework      | High     |
+
+
+## Step 2. Custom Model Type File
+Create definition file in your project repository that will describe my Custom Model Type:
+
+```bash
+  touch marketplace_builder/custom_model_types/todo.yml
+```
+
+## Step 3. Definition of Custom Model Type
+Using the editor of choice, add the definition of Custom Model Type.
+
+The code below is the description of properties that Customization will accept and the name of Custom Model Type object that is used to identify the object.
+For full list of Types that can be used see [CustomAttribute Types](/tutorials/custom-attributes/user-defined#customattribute-types)
+
+```yml
+name: todo
+properties:
+- name: task
+  type: string
+- name: priority
+  type: string
+```
+
+## Step 4. Sync/Deploy Custom Model Type definition.
+
+At the moment of deploy Marketplace-Kit is asking platformOS if Custom Model Type object with name `todo` already exists, if so it will be updated with given definition otherwise new object is created.
```

Comment:
```suggestion
At the moment of deploy marketplace-kit is asking platformOS if Custom Model Type object with name `todo` already exists, if so it will be updated with given definition otherwise new object is created.
```